### DISCUSSION TITLE
Restore split view

### DISCUSSION THREAD

**Discussion by h14turkiye** [zen-browser/desktop#2494] [Ideas]
Restore windows on startup is cool, there should be a configurable option to restore split views as well.

Lets say I am using 3 pinned tabs that is split, which in this case AI frontiers, and I close my window. When I start it up again I need to do the whole process of split viewing things again, which is not easy with keyboard only. 

![image](https://github.com/user-attachments/assets/01027847-35db-4a90-be41-39c2d01a4fe5)

This also would help me get something like Arc web browser's easel live capture, whenever I need to check news, mails, notifications from social media apps, I could easily just go to my pinned tabs and checkout everything in a second. 

Also I know you guys are cooking tab groups, which I hope reduces the space multiple tabs holds to one tab~, and also works with split view and pinned tabs.

**Comment by tedbergstrand** [comment]
Agreed. I think some sort of grouping for split tabs would be great. Both from a re-launching perspective as well as a tab management perspective. I also think that split tabs should naturally group together-- whether they move together in the sidebar or they enter some sort of "folder" type container to be treated as a single unit.  Hotkeys for Splits need a lot more love, too, as h14turkiye was saying.

**Comment by thesamim** [comment]
Surprised there's not more support for this.

**Comment by h14turkiye** [comment]
All of the things mentioned here are available in the recent version of Zen, I'm closing this as it is resolved.
